tasks/eth/022-holocene-fp-upgrade/.env

```
@@ -0,0 +1,11 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+OWNER_SAFE=0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A
+COUNCIL_SAFE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+
+# TO CONFIRM
+# SAFE_NONCE=6 # should not be necessary, next queued up task
+# SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=8 # current
+SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=11 # +1, task 021
```

Even though you are right it's not necessary to have these in since this is the next task for these safes, I would feel a bit better being explicit here, just in case

```suggestion
SAFE_NONCE=6
SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=8
SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=11
```